e2e/base/actions.spec.ts

```diff
@@ -1,5 +1,12 @@
 import { expect } from "@playwright/test";
 
+import {
+  Label as AppLabel,
+  TestId as AppTestId,
+} from "pages/EntityDetails/App/types";
+import { Label as ConfirmationDialogLabel } from "panels/ActionsPanel/ConfirmationDialog/types";
+import { TestId as ActionsPanelTestId } from "panels/ActionsPanel/types";
```

Comment:
We have to import the enums from the types files instead of the index, otherwise the index will try to load the component as well, which won't be built.